qa/rpc-tests/bigblocks.py

```diff
@@ -67,7 +68,10 @@ def setup_chain(self):
             for i in range(0,200):
                 miner = i%2
                 set_node_times(self.nodes, block_time)
-                self.nodes[miner].generate(1)
+                b1hash = self.nodes[miner].generate(1)[0]
+                b1 = self.nodes[miner].getblock(b1hash, True)
+                if miner % 2: assert(not (b1['version'] & FORK_BLOCK_BIT))
```

Comment:
Obviously there isn't an easier way to get if the nodes are voting for 2mb or not, but since order matters, you may want to either put that comment above the node creation, or above the if block for assert statements. Its a bit fragile to depend on the order, but it would be a pain to extend the proxy to give access to the expose the vote via rpc.